pkgs/development/libraries/http-parser/default.nix

```
@@ -12,7 +12,11 @@ stdenv.mkDerivation rec {
   };
 
   NIX_CFLAGS_COMPILE = "-Wno-error";
-  patches = [ ./build-shared.patch ];
+  patches = [ ./build-shared.patch ] ++ lib.optionals stdenv.isAarch32
+            [(fetchpatch {
+              url = "https://github.com/nodejs/http-parser/commit/4f15b7d510dc7c6361a26a7c6d2f7c3a17f8d878.patch";
```

Please also refer to the PR with a comment, this way reviewers can more easily validate the authenticity

```suggestion
              # https://github.com/nodejs/http-parser/pull/510
              url = "https://github.com/nodejs/http-parser/commit/4f15b7d510dc7c6361a26a7c6d2f7c3a17f8d878.patch";
```

---

Change made with 906c544. Many thanks!